### PR TITLE
fix build for Windows UWP

### PR DESCRIPTION
**Pull request by Rechi**

### CODE REVIEW

File: src/pvrclient-nextpvr.cpp [hunk 1]
```diff
@@ -147,19 +150,19 @@ cPVRClientNextPVR::~cPVRClientNextPVR()
...
     result.push_back(s);
     return result;
   }
-  CStdString::const_iterator substart = s.begin(), subend;
+  std::string::const_iterator substart = s.begin(), subend;
   while (true)
   {
     subend = search(substart, s.end(), delim.begin(), delim.end());
-    CStdString temp(substart, subend);
+	std::string temp(substart, subend);
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Indentation.

File: src/pvrclient-nextpvr.cpp [hunk 2]
```diff
@@ -200,34 +203,34 @@ bool cPVRClientNextPVR::Connect()
         XBMC->Log(LOG_DEBUG, "session.initiate returns: sid=%s salt=%s", m_sid, salt);
         
 
-        CStdString pinMD5 = PVRXBMC::XBMC_MD5::GetMD5(g_szPin);
-        pinMD5.ToLower();
+		std::string pinMD5 = PVRXBMC::XBMC_MD5::GetMD5(g_szPin);
+		StringUtils::ToLower(pinMD5);
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Indentation.

File: src/pvrclient-nextpvr.cpp [hunk 3]
```diff
@@ -200,34 +203,34 @@ bool cPVRClientNextPVR::Connect()
...
 
         // calculate combined MD5
-        CStdString combinedMD5;
+		std::string combinedMD5;
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Indentation.

File: src/pvrclient-nextpvr.cpp [hunk 4]
```diff
@@ -200,34 +203,34 @@ bool cPVRClientNextPVR::Connect()
...
         combinedMD5.append(":");
         combinedMD5.append(pinMD5);
         combinedMD5.append(":");
         combinedMD5.append(salt);
 
         // get digest
-        CStdString md5 = PVRXBMC::XBMC_MD5::GetMD5(combinedMD5);
+		std::string md5 = PVRXBMC::XBMC_MD5::GetMD5(combinedMD5);
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Indentation. ;-)

File: src/pvrclient-nextpvr.cpp [hunk 5]
```diff
@@ -200,34 +203,34 @@ bool cPVRClientNextPVR::Connect()
...
 
         // login session
-        CStdString loginResponse;
+		std::string loginResponse;
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: dto

File: src/pvrclient-nextpvr.cpp [hunk 6]
```diff
@@ -200,34 +203,34 @@ bool cPVRClientNextPVR::Connect()
...
         char request[512];
         sprintf(request, "/service?method=session.login&sid=%s&md5=%s", m_sid, md5.c_str());
         if (DoRequest(request, loginResponse) == HTTP_OK)
         {
-          if (strstr(loginResponse, "<rsp stat=\"ok\">"))
+          if (strstr(loginResponse.c_str(), "<rsp stat=\"ok\">"))
           {
             // check server version
-            CStdString settings;
+			std::string settings;
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: again, indentation

File: src/pvrclient-nextpvr.cpp [hunk 7]
```diff
@@ -1078,13 +1081,13 @@ int cPVRClientNextPVR::GetNumTimers(void)
 
 PVR_ERROR cPVRClientNextPVR::GetTimers(ADDON_HANDLE handle)
 {
-  CStdString response;
+	std::string response;
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: Indentation

File: src/pvrclient-nextpvr.cpp [hunk 8]
```diff
@@ -38,6 +39,8 @@
   #define MAXINT64 ULONG_MAX
 #endif 
 
+#include <algorithm>
```
Comment (ksooo) [inline src/pvrclient-nextpvr.cpp]: For what do you need this header?

Comment (Rechi) [reply src/pvrclient-nextpvr.cpp]: https://github.com/kodi-pvr/pvr.nextpvr/blob/0df87de8397c74f58e7371445677d088f67e2b55/src/pvrclient-nextpvr.cpp#L161

`algorithm` was included by `p8-platform/util/StdString.h`

File: src/pvrclient-nextpvr.cpp (hunk 1 above)
Comment (Rechi) [reply src/pvrclient-nextpvr.cpp]: fixed

File: src/pvrclient-nextpvr.cpp (hunk 8 above)
Comment (ksooo) [reply src/pvrclient-nextpvr.cpp]: I hate people putting "using namespace std;" into sources. :-/

**Pull request merged**